### DISCUSSION TITLE
Railings

### DISCUSSION THREAD

**Discussion by chrisanderso** [BentoGambin/Valheim-MissingPieces#5] [Ideas]
so i have a couple things I've ben wanting for a long time. first is hand railings . for both ladders and stairs. something nice and thin right now we have to use large angled beams not exactly elegant. the other thing is a spiral staircase that isn't more than 4m square . and maybe one that is half that size . but that might be asking a lot to try to walk up as a player .

**Comment by chrisanderso** [comment]
sorry i forgot to say that iron wood and stone options would be nice as well

**Comment by BentoGambin** [comment]
I don't plan on adding railings, sorry. And the spiral staircase is probably beyond my abilities.

**Reply by chrisanderso** [reply]
ok thanks . btw i doubled posted buy mistake sorry about that

**Comment by chrisanderso** [comment]
I wonder if it's possible to make a single step that we can stack up to get
the height that is desired. Then we can jus drop one rotate the next one
click and so on so we can get up one wall hight in 8 steps. If you fallow
me.
